Automattic/jetpack
# Update/installation error to only show on plugins page

## Proposed changes:

* Only show Jetpack installation error notice on core plugins page

### Other information:


## Jetpack product discussion

P2: pbNhbs-922-p2#comment-19812

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Not sure how to test this on a Jurassic Ninja site other than somehow purposely mess up the installation of a plugin, so these instructions will be for local environment
2. Checkout this branch
3. Go to `projects/plugins/jetpack/jetpack.php` and edit line 143 to be
```php
if ( ! is_readable( $jetpack_autoloader ) && is_readable( $jetpack_module_headings_file ) ) {
```
to get the notice to show
4. Now go to `/wp-admin/plugins.php` and make sure the notice is showing
![image](https://github.com/Automattic/jetpack/assets/65001528/f2fcd498-142d-4148-bbb3-706a34ee9d7f)
5. Go to any other `wp-admin` page (ex. `/wp-admin/plugin-install.php`) and make sure the notice does not show
![image](https://github.com/Automattic/jetpack/assets/65001528/96db554a-df77-4823-a220-4b8e2e6059e9)
6. Since the line of code exactly the same for all the plugins, I am not sure it is necessary to test all of them. However, if you would like to, go to each plugins main PHP file and search for the string `Your installation of` and alter the conditional to make the notice show. Make sure the plugin is active on your site and you should be able to test each plugin that way
